plugins/discovery-azure-classic/src/test/java/org/elasticsearch/discovery/azure/classic/AzureMinimumMasterNodesTests.java

```diff
@@ -42,52 +37,35 @@
         autoMinMasterNodes = false)
 public class AzureMinimumMasterNodesTests extends AbstractAzureComputeServiceTestCase {
 
-    public AzureMinimumMasterNodesTests() {
-        super(AzureComputeServiceTwoNodesMock.TestPlugin.class);
-    }
-
     @Override
     protected Settings nodeSettings(int nodeOrdinal) {
-        Settings.Builder builder = Settings.builder()
+        return Settings.builder()
                 .put(super.nodeSettings(nodeOrdinal))
                 .put("discovery.zen.minimum_master_nodes", 2)
-                // Make the test run faster
-                .put(ZenDiscovery.JOIN_TIMEOUT_SETTING.getKey(), "50ms")
-                .put(ZenDiscovery.PING_TIMEOUT_SETTING.getKey(), "10ms")
-                .put("discovery.initial_state_timeout", "100ms");
-        return builder.build();
+                .put("discovery.initial_state_timeout", "100ms")
```

Comment:
this is interesting - we explicitly check it's not there? https://github.com/elastic/elasticsearch/blob/master/test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java#L586

Reply:
Yes